google/resource_container_cluster.go

```diff
@@ -393,7 +412,13 @@ func resourceContainerCluster() *schema.Resource {
 }
 
 func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) error {
-	containerApiVersion := getContainerApiVersion(d, ContainerClusterBaseApiVersion, ContainerClusterVersionedFeatures)
+	_, isRegionalCluster := d.GetOk("region")
+	if isRegionalCluster {
```

Comment:
This block isn't necessary- you can just add the feature to the var as defined at the top of the file. The name of it should be the name of the field in the schema, so "region"

Reply:
Got it! Thanks.